code-security/admin_guide/scan-monitor/monitor-fix-issues-in-scan/monitor-fix-issues-in-scan.adoc

```diff
@@ -0,0 +1,371 @@
+//+
+//image::[width=600]
++
+The support for History and Traceability is currently only IaC resources, and the support for Errors is currently only available for packages.
+
+==== Fix Cart
+
+A Fix Cart displays the issues you choose to fix before creating a Pull Request.
+
+image::proj-10.png[width=800]
+
+See xref:fix-issues-in-a-scan-result.adoc[Fix Issues in Scan] to know more on how to add issues to a fix cart.
+
+[#filter-scan-results]
+==== Filter Scan Results
+
+Prisma Cloud enables you to filter your scan results across all code categories. You can filter your scan results across five default filters.
+
+* <<repositories-,Repositories>>
+* <<branch-,Branch>>
+* <<code-categories,Code Categories>>
+* <<issue-status,Issue Status>>
+* <<severities-,Severities>>
+
+[#repositories-]
+===== Repositories
+
+A list of integrated repositories.
+
+image::proj-14.png[width=800]
+
+[#branch-]
+===== Branch
+
+A list of the supported branches of a VCS branch scan. Currently, the repository’s default branch is selected by default and cannot be configured. This configuration is applicable for views - Overview, IaC Misconfiguration, Vulnerabilities, Secrets, Licenses, and Build Integrity.
+
+image::proj-15.png[width=800]
+
+[#code-categories]
+===== Code Categories
+
+A Category filters resources according to Build Integrity, Compute, Drift, General, IAM, Kubernetes, Licenses, Monitoring, Networking, Public, Secrets, Storage, and Vulnerabilities.
+During the time of repositories integration on Prisma Cloud Code Security, your defined Categories associated with the repositories also help with filters.
+
+image::proj-13.png[width=600]
+
+[#issue-status]
+===== Issue Status
+
+Status for each scanned repository is created based on the non-conformance to a policy. The repository status can be further filtered as Errors, Suppressed and Passed.
+
+image::proj-11.png[width=800]
+
+[cols="1,2", options="header"]
+|===
+
+|Status
+|Description
+
+|Error
+|A resource appears with an error status when it is non-conformant to a policy.
+
+|Passed
+|A resource that has conformant policies or may have a history of fixed errors.
+
+|Suppressed
+|A resource previously appeared with a non-conformant policy but is suppressed with a Suppress action. To suppress a non-conformant policy in a resource is when you absolve the scanned result with a definitive explanation indicating the non-conformance to be not problematic.
+
+|Fix Pending
+|A fix awaiting a PR merge in your VCS console.
+
+|===
+
+Your scanned resources appear on *Code Security > Projects* with an active Error filter by default. You can choose to add more filters or remove the Error filter.
+
+[#severities-]
+===== Severities
+
+A Severities indicates an impact on a non-conformant resource in your repository. Resources can be filtered as Critical,High, Medium, Low and Informational in severity.
+
+image::proj-12.png[width=800]
+
```

Comment:
There are more filters that should be documented - see here https://docs.bridgecrew.io/docs/projects_january23#additional-filters

Reply:
Added the additional filters.

Reply:
Added the additional filters.